### ISSUE TITLE
Add Prater overlay to the package avatar

### ISSUE THREAD

**Issue by alexpeterson91** [dappnode/DAppNodePackage-Lodestar-Prater#27]
Add Prater Overlay to the package avatar

**Comment by nflaig** [comment]
@alexpeterson91 about the avatar, which overlay should be use, the rhino? where can I find the overlay you used for the other CLs, the avatar should be quite easy to create once we have the overlay

**Comment by alexpeterson91** [comment]
This is a good question. That's what we'd been doing for a long time but under the false impression that the rhino logo was the prater logo not the staking launchpad logo for ETH in general. So we decided to change it. The first and only changed client so far is nimbus prater but since its logo is on an all color background the black logo and lettering won't work as an overlay on the 4 other CLs including lodestar since I've been prepping all logos for dark mode (making white backgrounds transparent and editing black lettering with white shadow/3D depth effect so they display legibly in day and night mode) night mode is finally gonna be released in this next core release and we just finished up testing it yesterdsy. Right now I'd say ignore it and I'll get it done later today for me (it's almost 3am) since I need to do it for all prater CL packages so everything is uniform. And looks clean together. If I don't have it done by Friday ping me because it's high priority not just for lodestar but for all the prater and goerli clients due to shapella on the 14th.
